### DISCUSSION TITLE
Problem with numpy

### DISCUSSION THREAD

**Discussion by emenod1** [anyoptimization/pymoo#473] [Q&A]
I am not sure if it's the right place to talk about this but there are some issues in the code  with numpy (the latest versions i think). For example, some algorithms like AGE use np.int which is deprecated, and when SingleObjectiveOutput is used, "if pf" should be "if pf is not None" to avoid errors.

**Comment by blankjul** [comment]
good catch.  can you create a PR to fix the two issues you have mentioned?

**Reply by emenod1** [reply]
Ok I add a third similar issue in CADASurvival for CTAEA algorithm and I create the PR

**Comment by blankjul** [comment]
I have just merged it. Thanks for your effort!
